matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -97,14 +97,20 @@ impl Default for WebRtcSocketConfig {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq)]
+pub enum Channel {
```

Comment:
Should probably be pub(crate)?

Reply:
Good idea - I will make it pub (crate) for now so that we won't have to worry about backwards compatibility - I can imagine that we might want to make it public later to represent some default data channel settings